inc/TryOn.php

```
@@ -3,50 +3,82 @@
+		return wp_login_url();
+	}
+
+	/**
+	 * Redirect to Try-On page after login if the parameter is set.
+	 *
+	 * @param string   $redirect the redirect URL.
+	 * @param \WP_User $user the logged in user.
+	 *
+	 * @return string Redirect URL.
+	 */
+	public function redirect_to_try_on( $redirect, $user ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)